index.rst

```diff
@@ -1,7 +1,5 @@
 :tocdepth: 2
 
-.. sectnum::
```

Comment:
Yes, it removes the section numbers from the web version. I had hoped that `only::` would fix that but `only::` seems to be broken in the sense that the `sectnum::` directive is triggered before filtering occurs (see e.g. sphinx-doc/sphinx#2150).